package.json

```diff
@@ -2,13 +2,15 @@
     "name": "MyProject",
     "private": true,
     "devDependencies": {
-        "@salesforce/eslint-config-lwc": "^0.3.0",
-        "@salesforce/lwc-jest": "^0.5.5",
-        "acorn": "^6.1.1",
-        "eslint": "^5.15.1",
-        "husky": "^1.3.1",
-        "lint-staged": "^8.1.5",
-        "prettier": "^1.16.4"
+        "@prettier/plugin-xml": "^0.7.2",
+        "@salesforce/eslint-config-lwc": "^0.5.0",
+        "@salesforce/lwc-jest": "^0.6.2",
+        "acorn": "^7.1.1",
+        "eslint": "^6.8.0",
+        "husky": "^4.2.3",
```

Comment:
This version of `husky` requires git version >=2.18
Also, while we currently use the `pre-commit` hook, there is also `pre-push` which might be a better choice for running the syntax linting and maybe even fire off tests locally prior to allowing push